components/Background/Board.jsx

```diff
@@ -0,0 +1,70 @@
+'use client'
```

Comment:
what is this?

Reply:
@nomadbitcoin , the 'use client' line at the beginning of the component is used to indicate that this code should be executed on the client side, i.e. in the browser, and not on the server. This declaration is specific to Next.js 13 and is used to distinguish between components that run on the client and those that are rendered on the server. 

If a component needs to use React hooks (useState, useEffect, etc.) or interact with the browser (such as accessing the DOM), it should be marked as a Client Component with 'use client'

Reply:
Nice, gotcha!